docs/user/reporting/automating-report-generation.asciidoc

```diff
@@ -4,6 +4,8 @@
 
 To automatically generate PDF and CSV reports, generate a POST URL, then submit an HTTP `POST` request using {watcher} or a script. Alternatively, use {kib} to generate reports on a recurring schedule and share them with a list of emails that you specify.
 
+NOTE: In 8.19.0, API keys are used to authenticate requests to generate reports. If you have a cross-cluster search environment and want to generate reports from remote clusters, you must have the appropriate cluster and index privileges on the remote cluster and local cluster. For example, if requests are authenticated with an API key, the API key requires certain privileges on the local cluster that contains the leader index, instead of the remote. For more information and examples, refer to {ref}/remote-clusters-cert.html#remote-clusters-privileges-ccr[Configure privileges for cross-cluster replication].
```

Comment:
```suggestion
NOTE: In 8.19.0, API keys are used to authenticate requests to generate reports. If you have a cross-cluster search environment and want to generate reports from remote clusters, you must have the appropriate cluster and index privileges on the remote cluster and local cluster. For example, if requests are authenticated with an API key, the API key requires certain privileges on the local cluster that contains the leader index, instead of the remote. For more information and examples, refer to {ref}/remote-clusters-privileges-cert[Configure roles and users for remote clusters].
```